.travis.yml

```diff
@@ -2,17 +2,20 @@ language: go
 go:
     - 1.6
     - 1.7
+    - 1.11
+    - 1.13
+    - 1.14
     - tip
 matrix:
     fast_finish: true
     allow_failures:
         - go: tip
 go_import_path: github.com/go-ldap/ldif
 install:
-    - go get gopkg.in/asn1-ber.v1
-    - go get gopkg.in/ldap.v2
+    - go get github.com/go-ldap/ldap/v3
     - go get code.google.com/p/go.tools/cmd/cover || go get golang.org/x/tools/cmd/cover
-    - go get github.com/golang/lint/golint || true
+    - go get golang.org/x/lint || true
```

Comment:
maybe use `go install golang.org/x/lint` instead here? Although go get seems to work fine locally.

Reply:
@johnweldon Got there in the end - new import path for go1.10 onwards golang.org/x/lint/golint is now enforced by the project

Reply:
Nice, thanks!